platform/security/handler/security-handler-guest/src/main/webapp/js/view/Login.view.js

```diff
@@ -31,31 +31,32 @@ define([
             'keypress #username': 'logInEnter',
             'keypress #password': 'logInEnter'
         },
-        logInEnter: function(e) {
+        logInEnter: function (e) {
             if (e.keyCode === 13) {
                 this.logInUser();
             }
         },
-        logInUser: function() {
+        logInUser: function () {
             var view = this;
 
             var prevUrl = decodeURI($.url().param('prevurl'));
 
             document.cookie = "JSESSIONID=;path=/;expires=Thu, 01 Jan 1970 00:00:00 GMT;secure";
 
             $.ajax({
-                type: "GET",
-                url: document.URL,
-                async: false,
-                beforeSend: function (xhr) {
-                    var base64 = window.btoa(view.$('#username').val() + ":" + view.$('#password').val());
-                    xhr.setRequestHeader ("Authorization", "Basic " + base64);
+                type: "POST",
+                url: "/services/login",
+                data: {
+                    "username": view.$('#username').val(),
```

Comment:
I'm not sure, not much different than what we had before (just it was base64 encoded).